test/state/precompiles.cpp

```diff
@@ -176,7 +158,7 @@ PrecompileAnalysis point_evaluation_analyze(bytes_view, evmc_revision) noexcept
 
 PrecompileAnalysis bls12_g1add_analyze(bytes_view, evmc_revision) noexcept
 {
-    static constexpr auto BLS12_G1ADD_PRECOMPILE_GAS = 500;
+    static constexpr auto BLS12_G1ADD_PRECOMPILE_GAS = 375;
```

Comment:
If we are not going to squash the commits before merging this change should go to different commit.

Reply:
Same applies to g2add, g2mul, pairing, g2 map analysis

Reply:
squashed.